sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleListCriteria.java

```diff
@@ -0,0 +1,70 @@
+package de.symeda.sormas.api.sample;
```

Comment:
`SampleListCriteria`'s properties are a subset of `SampleCriteria`'s. The introduction of an extra class here creates additional overhead to maintain or extend these criteria.

Reply:
`SampleListCriteria` it is used strictly for the sample list. It uses only 3 of the 23 members of `SampleCriteria`. I would say that it reduces the overhead :).